Makefile

```diff
@@ -1,6 +1,7 @@
 # Extract the version from package.json
 VERSION=$(shell $(CURDIR)/rpmversion.sh | cut -d - -f 1)
 RELEASE=$(shell $(CURDIR)/rpmversion.sh | cut -d - -f 2)
+RPMBUILD=$(shell which rpmbuild)
```

Comment:
It appears that this shouldn't be necessary. According to Brian something else is going on with the builders and on RHEL /bin is a hard-link to /usr/bin which I've verified on my system. So let's revert back to using `/usr/bin/rpmbuild` everywhere and I'll look at it separately.